src/app/clusters/operational-credentials-server/operational-credentials-server.cpp

```diff
@@ -433,8 +436,6 @@ OpCredsFabricTableDelegate gFabricDelegate;
 
 void MatterOperationalCredentialsPluginServerInitCallback(void)
 {
-    emberAfPrintln(EMBER_AF_PRINT_DEBUG, "OpCreds: Initiating OpCreds cluster by writing fabrics list from fabric table.");
-
```

Comment:
Don't want to keep this log line?  It's being deleted instead of replaced with ChipLog format.